OPCB/insurgency/common/client/mainLoop.sqf

```diff
@@ -7,22 +7,27 @@ _playergrp 		= grpNull;
 _HQstate 			= startLocation;
 
 
-[] spawn { 
+[] spawn {
+
+	scriptName "ins_AIhandler";
+ 
 	_aiTimer = time;
 	while { true } do {
 		//AI			
 		call aiSpawn;
 		//if (time - _aiTimer > 30) then { _aiTimer = time + (random 10) - 5; };
 		//call groupAI;
 		sleep 1;
-		if (((player distance2D startLocation) > 1000) && {(daytime < 17.5) && {daytime > 6.5}}) then {
+		if (((player distance2D startLocation) > 1000) && {(east knowsAbout player) > 1}) then {
```

Comment:
I have tried using the knowsAbout command for the Commander feature but it was giving me the value '4' every time the unit spawned, as if for some reason the AI needs to know the players before initializing their skills or etc.  
Does your script implement a workaround?

Reply:
No. Maybe something is interfering with it or you did something wrong